tests/test-bitrot.bats

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env bats
+
+#WARNING!
+#Be careful don't use ((, cause (( $status == pp )) && echo Really WRONG!
+#the issue is that (( 0 == letters )) is always true ... :(
```

Comment:
It's the default header for my bats tests. 

If you have the chance of having a typo with  (( $status == 0 )) and swap the 0 with a O or whatever string it will be always true. 

status=0; (( $status == pp )) && echo Really WRONG!

It's just another bash "bug" that I need to report some day (they will think on it as a feature...).  

PD:  (( )) in bash script is for testing integers, so I reckon strings decay to 0 in bash (( )).